pkgs/development/web/kore/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, callPackage, openssl }:
+
+let
+  source = callPackage ./source.nix { };
+in
+stdenv.mkDerivation {
+  name = "kore-${source.version}";
+
+  src = source.src;
+
+  buildInputs = [ openssl ];
+
+  preConfigure = ''
+    makeFlags="$makeFlags PREFIX=$out"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "An easy to use web application framework for C";
+    homepage = https://kore.io;
+    license = licenses.isc;
+    platforms = platforms.all;
+    maintainers = [ "John M. Harris, Jr. <johnmh@openblox.org>" ];
```

Comment:
Add yourself to https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix instead so that you can just do `maintainers = with maintainers; [ JohnMHarrisJr ];`